src/shape/arc.js

```diff
@@ -459,6 +513,53 @@ extend(ChartInternal.prototype, {
 			})
 			.merge(mainArc);
 
+		if ($$.hasMultiArcGauge()) {
+			arcLabelLines = main.selectAll(`.${CLASS.arcs}`)
+				.selectAll(`.${CLASS.arcLabelLine}`)
+				.data($$.arcData.bind($$));
+
+			mainArcLabelLine = arcLabelLines.enter()
+				.append("rect")
+				.attr("class", d => `${CLASS.arcLabelLine} ${CLASS.target} ${CLASS.target}-${d.data.id}`)
+				.merge(arcLabelLines);
+
+			if ($$.filterTargetsToShow($$.data.targets).length === 1) {
```

Comment:
IMO, when one data remains visible letting text label as is, make more sense than displaying at the center of gauge.
![image](https://user-images.githubusercontent.com/2178435/72120615-d7b2fb00-339b-11ea-9727-15431f60f2dc.png)